lookout/style/format/features.py

```diff
@@ -1,14 +1,61 @@
 """Features definition."""
+from collections import Counter
 from enum import Enum, unique
 import importlib
-from typing import (Callable, Iterable, List, Mapping, MutableMapping, Sequence,  # noqa: F401
-                    Tuple, Union)
+from typing import (Any, Generic, Iterable, List, Mapping, MutableMapping, Optional, Sequence,
+                    Tuple, TypeVar)
 
 import bblfsh
 from lookout.core.ports import Type
+import numpy
+from scipy.sparse import csr_matrix
 
 from lookout.style.format.classes import CLASSES
-from lookout.style.format.virtual_node import VirtualNode
+from lookout.style.format.virtual_node import AnyNode, VirtualNode
+
+FEATURES_NUMPY_TYPE = numpy.uint8
```

Comment:
just `FEATURES_TYPE` ?

Reply:
It's more informative this way I feel.

Reply:
Please comment again if you think there is an upside to renaming!